### ISSUE TITLE
Brain lesion

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7058]
**Preferred term label:**
Brain lesion

**Synonyms**
Cerebral lesion

**Definition (free text, please give PubMed ID)**
Focal pathological changes characterized by alteration in the size, shape and organization of the cellular components of the brain tissue.
PMID:10867785

**Parent term (use hpo.jax.org/app)**
abnormal nervous system morphology 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
The term 'brain lesion' is so nonspecific that it hardly adds more information than Abnormality of brain morphology (https://hpo.jax.org/app/browse/term/HP:0012443). Therefore, it is not good as a separate ontology term.
We would be happy to expand coverage for Alzheimer but would prefer to use specific terms (noting that some articles only provide nonspecific descriptions such as brain lesions, but this is not good phenotyping!)
For software that uses HPO, this is actually automatically implied because of the annotation to Neurofibrillary tangles for Alzheimer.
